internal/codegen/values_test.go

```diff
@@ -56,7 +56,7 @@ func TestValues(t *testing.T) {
 	vals := NewValues(context.Background(), sm)
 	assert.DeepEqual(t, &Values{
 		Git: &git{
-			Ref:           plumbing.NewBranchReferenceName("main").String(),
```

Comment:
should u maybe just rewrite this to test both with a table test or something or does it matter